src/Spout/Writer/XLSX.php

```diff
@@ -66,15 +66,24 @@ public function setShouldCreateNewSheetsAutomatically($shouldCreateNewSheetsAuto
     /**
      * Configures the write and sets the current sheet pointer to a new sheet.
      *
+     * @param string $sheetName The custom name of the sheet
      * @return void
      * @throws \Box\Spout\Common\Exception\IOException If unable to open the file for writing
      */
-    protected function openWriter()
+    protected function openWriter($sheetName = null)
```

Comment:
same thing: I would leave this function as is (which simplifies the interface) and move the logic in the sheet itself